js/stdlib/ts/index.ts

```diff
@@ -1,3 +1,4 @@
+export {ask, done, yesno} from './ask';
```

Comment:
Does `* as ask` work as well?

Reply:
I believe changing the above line to `export * as ask from './ask';` would make it so that developers would be typing things like `ask.ask`, `ask.done` and `ask.yesno`.

![image](https://user-images.githubusercontent.com/43425812/151685974-f3451b78-4dcc-4a12-bbeb-8e214940a6a7.png)

I'm simulating this with changes to the `node_modules` directory in my tutorial workspace, since I believe we don't have an easy way to see the behavior caused by this change other than a standard library publish, based on what Dan told me.

Reply:
Okay, @DanBurton please review :)

Reply:
I am in favor of `export * as ask from './ask'`, because I do want those things namespaced like that.